pkg/devfile/types.go

```diff
@@ -1,4 +1,4 @@
-package server
+package devfile
```

Comment:
@jhadvig What's you opinion on this move? We have also `helm`, `knative`, `terminal` on this level, so it might be ok?

Alternatively, we could move all these packages into a `devconsole` package (another time)? Wdyt? :man_shrugging: